extensions/sql/policy/store/src/main/java/org/eclipse/dataspaceconnector/sql/policy/store/SqlPolicyStoreServiceExtension.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright (c) 2022 ZF Friedrichshafen AG
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       ZF Friedrichshafen AG - Initial API and Implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.sql.policy.store;
+
+import org.eclipse.dataspaceconnector.spi.policy.store.PolicyStore;
+import org.eclipse.dataspaceconnector.spi.system.Inject;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtension;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
+import org.eclipse.dataspaceconnector.spi.transaction.TransactionContext;
+import org.eclipse.dataspaceconnector.spi.transaction.datasource.DataSourceRegistry;
+
+public class SqlPolicyStoreServiceExtension implements ServiceExtension {
```

Comment:
please add the `@Provides(PolicyStore.class)` annotation, to inform the dependency injection mechanism